Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Capacities.mo

```diff
@@ -1,38 +1,37 @@
 within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences;
 block Capacities
-  "Returns nominal and minimal stage capacities required for calculating operating and stage part load ratios"
+  "Returns design and minimal stage capacities for current and next available higher and lower stage"
 
   parameter Integer nSta = 3
     "Total number of stages";
```

Comment:
It would be better to differentiate chiller stage and plant stage. So that chiller stage can only reflect the number of chillers being operating and the plant stages could include the case when both chiller and WSE are operating?

Reply:
Chiller stage depends on the staging matrix, while WSE only is always chiller stage 0. Upper chiller stage indices can exist with or without a WSE enabled.

Reply:
So number of chillers in each stage is determined based on the staging matrix.